etree.go

```diff
@@ -46,6 +46,10 @@ type ReadSettings struct {
 	// false.
 	PreserveCData bool
 
+	// Changes the default behavior of replacing the value of existing attributes in CreateAttribute
+	// and instead add the attribute multiple times to the list.
```

Comment:
Update this comment to the following:
```
When an element has two or more attributes with the same name,
preserve them instead of keeping only one. Default: false.
```